public/scripts/ssooverride.js

```diff
@@ -4,14 +4,27 @@
  * the persona SSO that we employ for webmaker.org
  */
 (function() {
-  var hostname = document.getElementById("ssooverride").getAttribute("data-login-hostname");
+  var authHostname = document.getElementById("ssooverride").getAttribute("data-login-authHostname");
+  var thimbleHostname = document.getElementById("ssooverride").getAttribute("data-login-thimbleHostname");
   var loginToSave = document.getElementById("ssooverride").getAttribute("data-login-to-save");
+  var oauthClientId = document.getElementById("ssooverride").getAttribute("data-oauth-clientid");
 
-  require(["jquery", "thimblePage", "url-template"], function($, editor, urlTemplate) {
+  var user = {
+    username: document.getElementById("ssooverride").getAttribute("data-oauth-username"),
+    gravatar: document.getElementById("ssooverride").getAttribute("data-oauth-gravatar")
```

Comment:
would greatly prefer this to stay `avatar`. There is no reason to hardcode the branding.